pkg/artifactory/resource/replication/resource_artifactory_pull_replication_test.go

```diff
@@ -40,22 +42,70 @@ func mkTclForPullRepConfg(name, cron, url string) string {
 
 func TestAccPullReplicationInvalidCron(t *testing.T) {
 
-	_, fqrn, name := test.MkNames("lib-local", "artifactory_pull_replication")
+	_, _, name := test.MkNames("lib-local", "artifactory_pull_replication")
 	var failCron = mkTclForPullRepConfg(name, "0 0 * * * !!", acctest.GetArtifactoryUrl(t))
 
 	resource.Test(t, resource.TestCase{
 		PreCheck:          func() { acctest.PreCheck(t) },
 		ProviderFactories: acctest.ProviderFactories,
-		CheckDestroy:      testAccCheckReplicationDestroy(fqrn),
```

Comment:
Just to confirm for my own understanding: this step is removed because a replication is never created since the cron is invalid, right?